data/mitsjson.json

```diff
@@ -7,11 +7,11 @@
                 "sub_criteria": [
                     {
                         "type": "FGRCriterium",
-                        "fgrtype": "Getijdengebied"
```

Comment:
wat is de gedachte achter fgrtype => wanted_fgrtype?

Reply:
Omdat we het opgezochte type ook willen bewaren ben ik naar een wanted_fgrtype en een actual_fgrtype gegaan, ipv zoals eerder enkel fgrtype.